python/tests/test_pyvw.py

```diff
@@ -122,13 +122,19 @@ def test_multilabel_prediction_type():
 
 def test_cbandits_label():
     model = vw(cb=4, quiet=True)
-    assert pyvw.cbandits_label(model.example('1 |')).costs[0].label == 1
+    cbl = pyvw.cbandits_label(model.example('1 |'))
+    assert cbl.costs[0].label == 1
```

Comment:
There are 3 components to the label: cost, action and prob. Can you please test all 3?